### PR TITLE
Include description option in API for CRITS

### PR DESCRIPTION
**Pull request by mike-bailey**
May also wanna update the Wiki with the fix. I just implemented it how the rest of stuff is.

### CODE REVIEW

**Comment by frbapolkosnik** [comment]
Looks good to me

**Comment by mike-bailey** [comment]
Related to https://github.com/crits/crits/issues/784
I'm sure this could be done to other endpoints (for instance /ips), I just don't have time to do it to other ones and I needed the samples descrip ASAP for a project.

File: crits/samples/api.py [hunk 1]
```diff
@@ -83,6 +84,7 @@ def obj_create(self, bundle, **kwargs):
         related_type = bundle.data.get('related_type', None)
         backdoor_name = bundle.data.get('backdoor_name', None)
         backdoor_version = bundle.data.get('backdoor_version', None)
+	description = bundle.data.get('description', None)
```
Comment (frbapolkosnik) [inline crits/samples/api.py]: Is the indent off here?

**Comment by mike-bailey** [comment]
Crap sorry one second. Lemme check.

File: crits/samples/api.py (hunk 1 above)
Comment (mike-bailey) [inline crits/samples/api.py]: Seems so. One moment.

**Comment by mike-bailey** [comment]
Bear with me on the spacing issue, editor bug.

File: crits/samples/api.py (hunk 1 above)
Comment (mike-bailey) [inline crits/samples/api.py]: @frbapolkosnik good now?

**Comment by mgoffin** [comment]
This requires an update to:

https://github.com/crits/crits/blob/master/documentation/help_main.txt

and then `rst2html.py help_main.txt help_main.html` to update the API documentation.

**Comment by mike-bailey** [comment]
Sure. Also, it currently only does it if it's metadata. This worked for our use case but isn't the case if it's public. I'll fix that.

**Comment by mike-bailey** [comment]
Should be fixed.

File: crits/samples/api.py [hunk 2]
```diff
@@ -83,6 +84,7 @@ def obj_create(self, bundle, **kwargs):
         related_type = bundle.data.get('related_type', None)
         backdoor_name = bundle.data.get('backdoor_name', None)
         backdoor_version = bundle.data.get('backdoor_version', None)
+        description = bundle.data.get('description', None)
```
Comment (mgoffin) [inline crits/samples/api.py]: Wouldn't this just be a duplicate of the above description you've added?

Comment (mike-bailey) [inline crits/samples/api.py]: It's moved from the past commit, not copied. Unless I messed up somehow. Before it was under metadata, but it should be able to assign a descrip regardless.

Comment (mike-bailey) [inline crits/samples/api.py]: Ah you're right

**Comment by mike-bailey** [comment]
Sorry if this kinda shifted targets @mgoffin, but the same thing but with /api/v1/ips/ became a priority too so I went ahead and included it. Also fixed duplicate.

File: crits/samples/api.py (hunk 2 above)
Comment (mike-bailey) [inline crits/samples/api.py]: Fixed this in the previous commits in addition to changing IP.

**Pull request merged**